user-guide/Advanced_Functionality/Security/About_DMS_Security/Cloud_connectivity_and_security.md

```diff
@@ -0,0 +1,44 @@
+---
+uid: Cloud_connectivity_and_security
+---
+
+# Cloud connectivity and security
+
+From version 10.2.0 onwards, DataMiner comes bundled with a load of cloud features. Because the cloud means connecting your system to the dangers of the internet, we built security into the core design of our Cloud Connectivity framework.
```

Comment:
Maybe go for something a bit more formal than "a load of" – this phrasing doesn't seem very appropriate in this context.